main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
+genres = [
+    "Western",
+    "Action",
+    "Dramma",
+]
+
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson"),
+]
+
 
 def main() -> QuerySet:
-    pass
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for elem_first, elem_second in actors:
```

Comment:
You could use better variable names to indicate their content
```suggestion
    for first_name, last_name in actors:
```